src/couldBeSharedComponents/LayoutPanels/LayoutPanels.tsx

```diff
@@ -77,6 +77,11 @@ export const LayoutPanels = ({
 
   const isLeftPanelOpenToUse = isLeftPanelOpen ?? isLeftPanelOpenInternal;
   const isRightPanelOpenToUse = isRightPanelOpen ?? isRightPanelOpenInternal;
+  const [isLeftPanelContentShowing, setIsLeftPanelContentShowing] =
+    useState(isLeftPanelOpenToUse);
+  const [isRightPanelContentShowing, setIsRightPanelContentShowing] = useState(
+    isRightPanelOpenToUse,
+  );
   const setIsLeftPanelOpenToUse =
```

Comment:
I think a dev might have questions about what the difference between show hide state for the panel vs panel contents is and why it exists. Do the comments/self documenting below suffice or is more needed?